packages/opentelemetry-propagator-b3/test/B3MultiPropagator.test.ts

```diff
@@ -138,6 +139,23 @@ describe('B3MultiPropagator', () => {
       assert.deepStrictEqual(carrier[X_B3_FLAGS], undefined);
       assert.deepStrictEqual(carrier[X_B3_PARENT_SPAN_ID], undefined);
     });
+
+    it('should not inject if instrumentation suppressed', () => {
+      const spanContext = {
+        traceId: 'd4cda95b652f4a1592b449d5929fda1b',
+        spanId: '6e0c63257de34c92',
+        traceFlags: TraceFlags.SAMPLED,
+      };
+      b3Propagator.inject(
+        suppressInstrumentation(setSpanContext(ROOT_CONTEXT, spanContext)),
+        carrier,
+        defaultTextMapSetter
+      );
+      assert.deepStrictEqual(carrier[X_B3_TRACE_ID], undefined);
```

Comment:
```suggestion
      assert.strictEqual(carrier[X_B3_TRACE_ID], undefined);
```
same in the following lines